src/gennylib/src/PoolManager.cpp

```diff
@@ -53,9 +54,21 @@ auto createPool(const std::string& name,
 }  // namespace genny::v1
 
 
-mongocxx::pool::entry genny::v1::PoolManager::client(const std::string& name,
-                                                     size_t instance,
-                                                     const Node& context) {
+mongocxx::pool::entry genny::v1::PoolManager::create_client(const std::string& name,
+                                                            size_t instance,
+                                                            const Node& context) {
+    auto pool_entry = this->_create_client(name, instance, context);
+    bool prewarm = context["Clients"][name]["PreWarm"].maybe<bool>().value_or(true);
+    if (prewarm) {
+        pool_entry->list_databases();
```

Comment:
Regarding which command to use for pre-warming:

1. Both [ping](https://www.mongodb.com/docs/manual/reference/command/ping/) and [hello](https://www.mongodb.com/docs/manual/reference/command/hello/) are issued against a specific database, not the server. I could try creating a database to do this, but that might mess with tests which test creating a database that happens to use the same database name as the pre-warming code.
2. The pre-warming is run within Actor constructors, so it is completely outside any measurement code path.

Reply:
As discussed in Slack with @thessem , I've decided to pre-warm by pinging the admin DB instead.